### ISSUE TITLE
Group related form elements

### ISSUE THREAD

**Issue by adriannelee** [cds-snc/notification-planning#470]
API1Partially Supports1.3.1: Info and RelationshipsThe safelists for e-mails and phone numbers should be grouped into lists. In this way the grouping and positioning of the items can be done automatically and not have to be tracked with extra information in the labels. The remove button would also be included in the same list item to aid in the relationship with the input it affects. Although it would likely simplify the interaction and cognitive load if all of the inputs were always present.
https://dl.airtable.com/.attachments/5b9e8ae2b5803ab1a4357dc677777a17/5f0f6c7e/Capturedecranle2022-03-15a09_56_43.pngInteraction
